apps/bfd-pipeline/bfd-pipeline-app/src/main/java/gov/cms/bfd/pipeline/app/AppConfiguration.java

```diff
@@ -484,6 +508,26 @@ static ConfigLoader createConfigLoaderForTesting(Map<String, String> getenv) {
     return ConfigLoader.builder().addMap(DEFAULT_CONFIG_VALUES).addMap(getenv).build();
   }
 
+  /**
+   * Loads Backfill Configuration.
+   *
+   * @param config Config loader
+   * @param ccwPipelineEnabled True if the CCW pipeline is enabled in this instance.
+   * @return {@link BackfillConfigOptions}
+   */
+  public static BackfillConfigOptions loadBackfillConfigOptions(
+      ConfigLoader config, boolean ccwPipelineEnabled) {
+    boolean enabled = config.booleanOption(SSM_PATH_SAMHSA_BACKFILL_ENABLED).orElse(false);
+    // We don't want to run if we're on a CCW Pipeline instance
+    if (!enabled || ccwPipelineEnabled) {
```

Comment:
It is a bit confusing that this runs on the RDA pipeline, but I understand the reasoning. Ideally this could run as its own pipeline, but that would increase the complexity a fair bit. I think this is fine for now and we can revisit once we're running in ECS.